### PR TITLE
EWL-4169: Migrate over headings

### PR DESCRIPTION
**Pull request by nstriedinger**
**Jira Ticket**
- https://issues.ama-assn.org/browse/EWL-4169

## Description

Migrate over h(1-6) headings. Fix link.twig and un-nest unnecessary nesting.

## To Test

- [ ] Get Styleguide 2 up and running
- [ ] Click on http://localhost:3000/?p=atoms-headings
- [ ] Verify that headings are the correct size on Mobile, Tablet, and Desktop.

## Visual Regressions

Does your work introduce any visual regressions to existing work in the Style Guide, or in a Drupal 8 environment? Please either call these out here or address them before marking your PR as `ready for review`.

## Relevant Screenshots/GIFs

Use something like [Skitch](https://evernote.com/skitch/) or [GIPHY Capture](https://giphy.com/apps/giphycapture) to capture images/gifs to demonstrate behaviors.

## Remaining Tasks

Remaining tasks?

## Additional Notes

Anything more to add? Good things to call out here are:
- are there any PRs in this or other repositories that relate to or affect this PR?
- are there any caveats or oddities testers should know about when testing this PR?
- are there any problems you ran into during your work that you'd like to call out?

### CODE REVIEW

File: styleguide/source/assets/scss/00-base/_breakpoints.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// Breakpoints
+$bp-xs: 25em; // 400px
```
[the remark this answers is not in the document]
Comment (nstriedinger) [reply styleguide/source/assets/scss/00-base/_breakpoints.scss]: @designerneil -- I'd prefer rems, personally.

[the remark this answers is not in the document]
Comment (designerneil) [reply styleguide/source/assets/scss/00-base/_breakpoints.scss]: ok. let's go with rems

File: styleguide/source/_patterns/01-atoms/heading.twig [hunk 2]
```diff
@@ -0,0 +1 @@
+<h{{ heading.level }} class="{{ heading.class }}">{{ heading.text }}</h{{ heading.level }}>
```
[the remark this answers is not in the document]
Comment (nstriedinger) [reply styleguide/source/_patterns/01-atoms/heading.twig]: Done.

[the remark this answers is not in the document]
Comment (designerneil) [reply styleguide/source/_patterns/01-atoms/heading.twig]: I feel this is weird when it's broken up like this h{{heading.level}}
Can we just do {{heading.level}}??

File: styleguide/source/_patterns/01-atoms/heading.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "heading": {
+      "level": "1",
```
[the remark this answers is not in the document]
Comment (nstriedinger) [reply styleguide/source/_patterns/01-atoms/heading.json]: @designerneil -- nope, this looks good as-is. we don't need the repetitive "h".

File: styleguide/source/_patterns/01-atoms/heading.twig [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<h{{ heading.level }} class="{{ heading.class }}">
+{{ heading.text }}
```
Comment (designerneil) [inline styleguide/source/_patterns/01-atoms/heading.twig]: indent here

**Pull request merged**